test/adapters/file/file.spec.js

```diff
@@ -10,7 +10,7 @@ var expect_to_fail = juttle_test_utils.expect_to_fail;
 
 var tmp_file = tmp.tmpNameSync();
 
-var validFormats = {
+var symmetricalFormats = {
```

Comment:
Oh right -- now I remember commenting on this in the other file.

In context the change makes some sense, but then I would think you'd want a test for grok format in the file adapter here to make it consistent and to actually test that grok is usable in files.